starknet_py/contract.py

```diff
@@ -415,6 +480,28 @@ async def from_address(
 
         return Contract(address=address, abi=abi, client=client)
 
+    @staticmethod
+    async def declare(
+        account: AccountClient,
+        compiled_contract: str,
+        max_fee: Optional[int] = None,
+        auto_estimate: bool = False,
+    ) -> DeclareResult:
```

Comment:
```suggestion
    async def declare(
        account: AccountClient,
        compiled_contract: str,
        *,
        max_fee: Optional[int] = None,
        auto_estimate: bool = False,
    ) -> DeclareResult:
```
What about this? I like the idea of enforcing optional parameters to be passed as keyword arguments, but it's perhaps inconsistent with the rest of the `Contract`.